src/entities/college.entity.ts

```diff
@@ -0,0 +1,43 @@
+import { 
+  Entity, 
+  PrimaryGeneratedColumn, 
+  Column, 
+  CreateDateColumn, 
+  UpdateDateColumn, 
+  DeleteDateColumn, } from "typeorm";
+
+@Entity()
+export class College {
+  @PrimaryGeneratedColumn("uuid")
+  id!: string;
+
+  @Column({ type: "varchar", nullable: false })
+  name!: string;
+
+  @Column({ type: "varchar", nullable: true })
+  uniform?: string;
+
+  @Column({ type: "date", nullable: false })
```

Comment:
there should be a name parameter in the column. name: "year_established"...and when you say year established. are you only interested in storing the year. what of month and date